common/eth2/src/lib.rs

```diff
@@ -1044,6 +1044,24 @@ impl BeaconNodeHttpClient {
         Ok(())
     }
 
+    /// `POST beacon/rewards/attestations`
+    pub async fn post_beacon_rewards_attestations(
+        &self,
+        attestations: &[lighthouse::StandardAttestationRewards],
```

Comment:
This should be a `&[ValidatorId]`

It would be great to have an automated test that calls this, but I realise we're running out of time so I can come back for that later.